Dockerfile

```diff
@@ -61,6 +61,10 @@ RUN install -o daemon -g daemon -d "${IMS_SERVER_ROOT}"
 # Copy build result
 COPY --from=build "${IMS_INSTALL_DIR}" "${IMS_INSTALL_DIR}"
 
+RUN python -m pip --no-cache-dir install pipx
+RUN pipx install tox
+#RUN tox deps
+
```

Comment:
@toddgardner is this required? 

I think this is the same `Dockerfile` that is used in production, so I suggest if we need to make changes we make a duplicate something like `dev.Dockerfile` Unless you think that change is needed in production too.

Reply:
Ah no sorry bad merge. I've been trying to fix the lint issues, and I can't get tox running locally either in docker or out of it, this was caught up in a bad push.

Reply:
Fixed